.dotnet/example_code/DynamoDB/TryDax/05-Scan-Test.cs

```diff
@@ -25,28 +25,19 @@
 using Amazon.Runtime;
 using Amazon.DAX;
 using Amazon.DynamoDBv2.Model;
-using System.Collections.Generic;
-using System;
-using Amazon.DynamoDBv2;
-using Amazon;
 
 namespace ClientTest
 {
     class Program
     {
         static void Main(string[] args)
         {
+            string endpointUri = args[0];
+            Console.WriteLine("Using DAX client - endpointUri=" + endpointUri);
```

Comment:
Use interpolated strings as described above.

Reply:
Revised as per discussion to match approved samples in TryDax.zip.